synapse/storage/roommember.py

```diff
@@ -74,6 +76,50 @@ def __init__(self, db_conn, hs):
         self._check_safe_current_state_events_membership_updated_txn(txn)
         txn.close()
 
+        if self.hs.config.metrics_flags.known_servers:
+            self._known_servers_count = 1
+            self.hs.get_clock().looping_call(self._count_known_servers, 60 * 1000)
```

Comment:
can you wrap _count_known_servers with a `run_as_background_process` to avoid warnings about running db txns from the sentinel context?